src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
```

Comment:
```suggestion
```

Reply:
?

Reply:
redundant blank line
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code